packages/clerk-js/src/ui/utils/colors.ts

```
@@ -255,6 +255,8 @@ const hslaColorToHslaString = ({ h, s, l, a }: HslaColor): HslaColorString => {
 };
 
 const parse = (str: string): ParsedResult => {
+  // TODO(Colors): This is a temporary fix to allow for custom colors to be passed in as variables
+  return str as unknown as ParsedResult;
   const prefix = str.substr(0, 3).toLowerCase();
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)